Another projekts/calculator/main_calculator_2.py

```diff
@@ -3,6 +3,7 @@
 print("Our Magical Calculator")
 print("Type 'quit' to exit\n")
 
+
```

Comment:
Please delete the extra line.

Reply:
Can you **replace** duplicated code 
```
            first_number = int(input("Enter first number: "))
            second_number = int(input("Enter second number: "))
            result = first_number + second_number
            print(f"Answer is {result}")
            user_input = input("Do you want to continue? Enter Yes or No: ")
            user_input = user_input.capitalize()
            if user_input == 'Yes':
                new = True
            else:
                break
```
     
with some `operate(param1 param2)` function?

Reply:
Could you apply this:
```
            if user_input == 'Yes':
                new = True
            else:
                break
```

To the calculator 2.